parfive/downloader.py

```diff
@@ -687,7 +687,7 @@ async def _http_download_worker(self, session, url, chunksize, http_range, queue
                 resp.headers,
             )
 
-            if resp.status != 200:
+            if resp.status < 200 or resp.status >= 300:
```

Comment:
Do we not want to exclude 300?
```suggestion
            if resp.status < 200 or resp.status > 300:
```

Reply:
We do, I just fail at logic apparently.